src/amo/css/inc/mixins.scss

```diff
@@ -15,75 +15,83 @@
 
 /* Font mixins. Changes here affect *all* apps */
 @mixin font-light() {
-  font-family: 'Fira Sans', sans-serif;
+  font-family: 'Inter', sans-serif;
```

Comment:
Is it possible to make these three lines into a mixin so that they don't have to be duplicated inside the definition of each font mixin below?

Reply:
done https://github.com/mozilla/addons-frontend/pull/10110/commits/6ccb26ab7aadde78f42accb1f9e0157ef384e881